dashboard-notifications/src/main/java/uk/gov/hmcts/reform/dashboard/services/DashboardNotificationService.java

```diff
@@ -53,6 +54,7 @@ public List<Notification> getNotifications(String ccdCaseIdentifier, String role
             .findByReferenceAndCitizenRole(ccdCaseIdentifier, roleType);
 
         return dashboardNotificationsEntityList.stream()
+            .sorted(Comparator.comparing(t -> t.getDashboardNotificationsTemplates().getCreatedAt(), Comparator.reverseOrder()))
```

Comment:
order should be against notification entity

Reply:
it's updated